configs/machines/generic.yaml

```diff
@@ -1,44 +1,158 @@
+#######################################################################################
+
 # GENERIC YAML CONFIGURATION FILES
+#
+# Set hostname of the machine 
+name: <hostname>
 
-name: generic
-accounting: false
-hetjob_flag: "?" # either hetjob or packjob
-batch_system: "slurm"
+# General information about operating system
+# Operating system
+#operating_system: "<your-operating-system>"
+# Shell interpreter (e.g. "/usr/bin/bash")
+sh_interpreter: "<path-to-shell>"
 
-jobtype: compute
+# Information about the job scheduler
+# Set batch system (e.g. slurm)
+batch_system: "<your-job-scheduler>"
 
-partition: generic 
+# Set whether an account needs to be set for batch system (e.g. true or false). The actual account that will be used for the experiment will be set in the runscript.
+accounting: true 
+#
+#hetjob_flag: "?" # either hetjob or packjob
+#
+#jobtype: compute
+#
+# Information about the partitions of the machine
+#
+# Specify if hyperthreading should be used.
+use_hyperthreading: False
+# Set further options depending on hyperthreading variable 'use_hyperthreading' by uncommenting and editing the following 'choose_use_hyperthreading' section:
+#choose_use_hyperthreading:
+#        True: 
+#            hyperthreading_flag: ""
+#            mt_launcher_flag: ""
+#        False: 
+#            <set_any_variable_here_related_to_hypertherading>: <value>
+#        "anyother_scalar_value':
+#            <set_any_variable_here_related_to_hypertherading>: <value>
 
+# Set default partition ??? (e.g. compute)
+partition: <partition_name>
+#
+# Set detail information for available partitions on the machine.
+# These variables will be used in other parts of this yaml file.
 choose_partition:
-        generic:
-                partition_name: generic
-                partition_cpn: 8 
-
+        <partition_name>:
+                partition_name: "<partition_name>"
+                partition_cpn: <number_of_cores_per_node>  # integer number 
+#
+# Define available partitions of the machine for different jobtypes (e.g. for computing [compute], post-processing [PP], etc.)
+#
 partitions:
         compute:
-                name: ${computer.partition_name}
+                name: ${computer.partition_name}     # this uses the variable set in choose_partition section (see above)
+                #name: "compute"                     # or set the partition name as string
                 cores_per_node: ${computer.partition_cpn}
-
-logical_cpus_per_core: 2
-
-threads_per_core: 1
-
+#
+#logical_cpus_per_core: 2
+#
+#threads_per_core: 1
+#
+# Specify different pool directories for this machine.
+#
 pool_directories:
-        focipool: "/path/not/set/for/generic/yaml/"
-        pool: "/path/not/set/for/generic/yaml"
+#        focipool: "<set_path_to_pool_dir>"
+#        pool: "<set_path_to_pool_dir"
+#
+# Is this a dublicate ??? Can be set in runscript.
+pool_dir:  "<set_path_to_pool_dir>"
```

Comment:
No, that's not a duplicate. Setting it in the runscript would override the one that is set for you by default here.

Reply:
Ok. Thanks for clarification. I will remove my comment.